.ci/generate_bear_requirements.py

```diff
@@ -151,6 +153,8 @@ def _get_requirements(requirements, default_operator, exclude=[]):
 def get_gem_requirements(requirements):
     return _get_requirements(requirements, '~>')
 
+def get_r_requirements(requirements):
+    return _get_requirements(requirements, '>=')
```

Comment:
That default isnt used anywhere here, because both bears add a comparison operator before the version.

i.e. `version='>=1.0.2'`  and `'>1.5'`

anyway, it is approved.  more tasks will be done later to make all this more useful.  It currently doesnt do anything, so cant break anything